content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -534,10 +534,10 @@ processing its startup data, you might prefer a readiness probe.
 
 {{< note >}}
 If you want to be able to drain requests when the Pod is deleted, you do not
-necessarily need a readiness probe; on deletion, the Pod automatically puts itself
-into an unready state regardless of whether the readiness probe exists.
-The Pod remains in the unready state while it waits for the containers in the Pod
-to stop.
+necessarily need a readiness probe; when the POD is deleted, the corresponding endpoint
+will have its `ready` status as `false`, so load balancers will not use it for regular
+traffic. The endpoint remains in the unready state while it waits for the containers
+in the Pod to stop.
```

Comment:
Maybe we should talk about [`serving`](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#serving) and [`ready`](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#ready) - or hyperlink to the relevant existing docs?

Reply:
I'll try to add details about endpoint conditions.